terraform/azure.tf

```diff
@@ -500,6 +500,68 @@ resource "azurerm_sql_database" "sql_database" {
   depends_on          = ["azurerm_sql_server.sql_server"]
 }
 
+
+resource "random_string" "mysql_server" {
+  length  = 10
+  special = false
+  upper   = false
+}
+
+resource "azurerm_mysql_server" "mysql" {
+  name                = "${random_string.mysql_server.result}-mysqlsvr"
+  location            = "westeurope"
+  resource_group_name = "${azurerm_resource_group.rg.name}"
+
+  sku {
+    name     = "B_Gen5_2"
+    capacity = "2"
+    tier     = "Basic"
+    family   = "Gen5"
+  }
+
+  storage_profile {
+    storage_mb            = "5120"
+    backup_retention_days = "7"
+    geo_redundant_backup  = "Disabled"
+  }
+
+  administrator_login          = "inspec-azure"
+  administrator_login_password = "P4assw0rd!"
+  version                      = "5.7"
+  ssl_enforcement              = "Enabled"
+}
+
+resource "azurerm_mysql_database" "mysql" {
```

Comment:
Hi, @mikaelkrief I have just noticed that you have 2 `azurerm_mysql_server ` resources named the same `mysql`. In fact, the below `azurerm_mysql_firewall_rule` resource has also been names `mysql`. Could you update the naming of these resources please, as currently, the terraform will fail to run?